system-tests/services/test-apps.js

```diff
@@ -100,14 +125,81 @@ describe("Services", function() {
         .contains(serviceName, { timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT })
         .should("exist");
 
-      // Wait for the table and the service to appear
+      // Now click on the name
       cy
         .get(".page-body-content table")
         .getTableRowThatContains(serviceName)
+        .get("a.table-cell-link-primary")
+        .contains(`${serviceName}`)
+        .click();
+
+      // open edit screen
+      cy
+        .get(".page-header-actions .dropdown")
+        .click()
+        .get(".dropdown-menu-items")
+        .contains("Edit")
+        .click();
+
+      // check if values are as expected
+      cy
+        .root()
+        .getFormGroupInputFor("Service ID *")
+        .should("have.value", `/${Cypress.env("TEST_UUID")}/${serviceName}`);
+
+      // TODO: Due to a bug in cypress you cannot type values with dots
+      // cy
+      //   .getFormGroupInputFor('CPUs *')
+      //   .contents('equal', '0.5');
+
+      cy
+        .root()
+        .getFormGroupInputFor("Memory (MiB) *")
+        .should("have.value", "10");
+
+      // Test Mesos Runtime again? should be tested before...
+    });
+
+    it("should fail create the same app name again", function() {
+      // same as above
+      const serviceName = "app-with-inline-shell-script";
+      const cmdline = "while true; do echo 'test' ; sleep 100 ; done";
+
+      // Select 'Single Container'
+      cy.contains("Single Container").click();
+
+      // Fill-in the input elements
+      cy
+        .root()
+        .getFormGroupInputFor("Service ID *")
+        .type(`{selectall}{rightarrow}${serviceName}`);
+
+      cy.root().getFormGroupInputFor("Command").type(cmdline);
+
+      // Select mesos runtime
+      selectMesosRuntime();
+
+      // Check JSON view
+      cy.contains("JSON Editor").click();
+
+      // Click Review and Run
+      cy.contains("button", "Review & Run").click();
+
+      // Run service
+      cy.contains("button", "Run Service").click();
+
+      // Also no Error should exist
+      cy
+        .wait("@appsReq")
+        .wait(2000)
```

Comment:
ok, lets try without it, if tests are passings thats okay for me. but we have to keep that in mind when cypres starts to fail randomly again